### PR TITLE
Created BaseTextArea component

### PR DESCRIPTION
**Pull request by usrj03l**

### CODE REVIEW

File: src/components/BaseTextArea/index.tsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  return (
+    <>
+      <textarea
+        onChange={(e) => Props.handleChange(e.target.value)}
+        value={Props.Value}
+        className={`${Props.className} 
+            h-[${Props.height}px] 
+            ${styles} 
+            ${Props.textError ? 'ring-red-500' : ''}`}
+      ></textarea>
```
Comment (alexrobaina) [inline src/components/BaseTextArea/index.tsx:23]: Can you change this line for /> 
It is not necessary to close with </textarea>

**Review by alexrobaina — COMMENTED** [review]
Hi @usrj03l I checked the PR and made several comments. 
Can you check my comments? 

Thank you

**Comment by usrj03l** [comment]
i have made all the suggested changes.

**Comment by alexrobaina** [comment]
Hi @usrj03l 
Thank you for your contribution.

**Pull request merged**